e2e/interchaintestv8/e2esuite/light_clients.go

```diff
@@ -39,6 +40,7 @@ func (s *TestSuite) createEthereumLightClient(
 	cosmosChain *cosmos.CosmosChain,
 	simdRelayerUser ibc.Wallet,
 	ibcContractAddress string,
+	rustFixtureGenerator *types.RustFixtureGenerator,
```

Comment:
should this be called the `ethFixtureGenerator`?

Reply:
Not sure if that makes sense. The Solidity one is also for Eth. `ethLightClientFixtureGenerator`? xD

Reply:
Oh sorry, it should have been `wasmFixtureGenerator`

Reply:
Fair enough!